app/src/main/java/com/kabouzeid/gramophone/dialogs/SleepTimerDialog.java

```diff
@@ -153,10 +200,22 @@ private PendingIntent makeTimerPendingIntent(int flag) {
     }
 
     private Intent makeTimerIntent() {
+        if (checkBox.isChecked()) {
+            return new Intent(getActivity(), MusicService.class)
```

Comment:
Nitpick: it'd look a little less redundant if you just have a variable assignment for the action.